applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java

```diff
@@ -566,30 +565,4 @@ private void maybeDestroyCurrentAd()
 
         clickableViews.clear();
     }
-
-    private void setImageUrl(final String imageUrl, final ImageView imageView)
```

Comment:
Did you auto-optimize imports as well?

Reply:
Oh, my bad.  Forgot to run optimize import.  Will push this now.